resources/assets/js/videos/components/videoScreen/annotationPlayback.vue

```diff
@@ -135,7 +136,13 @@ export default {
             source.addFeature(newFeature);
         },
         createFeature(annotation) {
```

Comment:
In which case is `createFeature()` called with an annotation that already exists as a feature? I feel that the method should not be called in the first place in such a case.